### PR TITLE
Make opening timestamp non-optional

### PR DESCRIPTION
**Pull request by kongzii**

### CODE REVIEW

File: prediction_market_agent_tooling/markets/omen/data_models.py [hunk 1]
```diff
@@ -118,12 +123,25 @@ class OmenMarket(BaseModel):
     resolutionTimestamp: t.Optional[int] = None
     answerFinalizedTimestamp: t.Optional[int] = None
     currentAnswer: t.Optional[str] = None
-    creationTimestamp: t.Optional[int] = None
+    creationTimestamp: int
```
Comment (kongzii) [inline prediction_market_agent_tooling/markets/omen/data_models.py:126]: Typed as non-optional on graph:

Screenshot by Dropbox Capture

File: prediction_market_agent_tooling/markets/omen/omen_subgraph_handler.py [hunk 2]
```diff
@@ -149,7 +160,9 @@ def _build_where_statements(
             where_stms["creationTimestamp_gt"] = to_int_timestamp(created_after)
 
         if opened_before:
-            where_stms["openingTimestamp_lt"] = to_int_timestamp(opened_before)
+            where_stms["question_"]["openingTimestamp_lt"] = to_int_timestamp(
```
Comment (evangriffiths) [inline prediction_market_agent_tooling/markets/omen/omen_subgraph_handler.py:163]: How come you've changed the 'where statements'?

Comment (kongzii) [reply prediction_market_agent_tooling/markets/omen/omen_subgraph_handler.py:163]: Btw I also tried to change it in https://github.com/gnosis/prediction-market-agent-tooling/blob/7906ccdb5a91f14bdeefe5ee0f67892ef6d3deab/prediction_market_agent_tooling/markets/omen/omen_subgraph_handler.py#L218

But I tried the following syntaxes:

- `self.trades_subgraph.FixedProductMarketMaker.question.openingTimestamp`
- `self.trades_subgraph.FixedProductMarketMaker.question__openingTimestamp`

And it doesn't work, maybe @gabrielfior knows?

Comment (gabrielfior) [reply prediction_market_agent_tooling/markets/omen/omen_subgraph_handler.py:163]: My only 2cents are that `fpmm.openingTimestamp` can be null, whereas `question.openingTimestamp` cannot (determined that querying the subgraph manually).
So I'm fine with this change.

Comment (kongzii) [reply prediction_market_agent_tooling/markets/omen/omen_subgraph_handler.py:163]: @gabrielfior That's cool, but do you know how that syntax I asked about should be written?

**Pull request merged**